docs/networking/deep-dive.md

```diff
@@ -0,0 +1,112 @@
+---
+sidebar_position: 3
+sidebar_label: Deep Dive
+title: ""
+keywords:
+- Harvester
+- Networking
+- Topology
+---
+# Harvester Network Deep Dive
+
+The network topology below reveals how we implement the Harvester network.
+
+![](/img/v1.1/networking/topology.png)
+
+The diagram contains [the built-in cluster network mgmt](./clusternetwork.md/#built-in-cluster-network) and a [custom cluster network](./clusternetwork.md/#custom-cluster-network) called `oob`.
+
+As shown above, the Harvester network primarily focuses on layer 2. We leverage Linux network devices to construct traffic paths for the communication between VM to VM, VM to host, and VM to external network devices.
+
+The Harvester network is divided into three layers, including:
+
+- KubeVirt networking layer
+
+- Harvester networking layer
+
+- external networking layer
+
+## Kubevirt Networking
+
+The basic idea of KubeVirt is to put libvirt into the Kubernetes pod. The KubeVirt network is responsible for forwarding traffic between the pod and VM. 
```

Comment:
`The basic idea of KubeVirt is to put libvirt into the Kubernetes pod.`  -> `The general purpose of KubeVirt is to run VM inside the Kubernetes pod. `

`The KubeVirt network is responsible for forwarding traffic between the pod and VM. ` -> `The KubeVirt network builds the network path between the pod and VM. `